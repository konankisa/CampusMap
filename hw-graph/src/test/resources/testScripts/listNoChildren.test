# Creates a graph, adds a node, and
# returns a list of the children

# Create a graph
CreateGraph graph

# Add a node to graph
AddNode graph node1

# List the children of the parent node
ListChildren graph node1