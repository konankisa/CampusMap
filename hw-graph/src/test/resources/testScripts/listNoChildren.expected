# Creates a graph, adds a node, and
# returns a list of the children

# Create a graph
created graph graph

# Add a node
added node node1 to graph

# List the children of the parent node
the children of node1 in graph are: